test_vm/src/util.rs

```diff
@@ -472,21 +472,28 @@ pub fn submit_windowed_post(
     };
     apply_ok(v, worker, maddr, TokenAmount::zero(), MinerMethod::SubmitWindowedPoSt as u64, params);
     let mut subinvocs = None;
-    if new_power != PowerPair::zero() {
-        let update_power_params = serialize(
-            &UpdateClaimedPowerParams {
-                raw_byte_delta: new_power.raw,
-                quality_adjusted_delta: new_power.qa,
-            },
-            "update claim params",
-        )
-        .unwrap();
-        subinvocs = Some(vec![ExpectInvocation {
-            to: *STORAGE_POWER_ACTOR_ADDR,
-            method: PowerMethod::UpdateClaimedPower as u64,
-            params: Some(update_power_params),
-            ..Default::default()
-        }]);
+    let invocs = v.take_invocations();
+    let last_invoc = invocs.last().unwrap();
+    if new_power.is_some() {
+        let new_pow = new_power.unwrap();
+        if new_pow == PowerPair::zero() {
+            assert_eq!(0, last_invoc.subinvocations.len())
```

Comment:
This works but it's more idiomatic to make subinvocs Some(vec![]) in this case

Reply:
Aha! Thank you. Changes are up :)